aries-site/src/examples/templates/dashboards/components/Measure.js

```diff
@@ -0,0 +1,34 @@
+import { cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Button, NameValuePair, Text } from 'grommet';
+
+export const Measure = ({ name, value, onClick }) => {
+  const { icon: iconProp, label } = name;
+  const icon = cloneElement(iconProp, { size: 'small' });
+
+  return (
+    <Button onClick={onClick}>
+      <NameValuePair
```

Comment:
Did you have a chance to see how this was in the screen reader?

Reply:
Not yet.

Reply:
okay no worries its on my check list